src/http/feed-http.js

```diff
@@ -2,11 +2,13 @@ import _ from 'lodash';
 import * as feedCore from '../core/feed-core';
 import {createJsonRoute, throwStatus} from '../util/express';
 import {assert} from '../validation';
+import CONST from '../constants';
 
 const getFeed = createJsonRoute(function(req, res) {
   const feedParams = assert({
     beforeId: req.query.beforeId,
-    limit: req.query.limit
+    limit: req.query.limit,
+    sort: req.query.sort || CONST.FEED_SORT_TYPES.NEW,
```

Comment:
I think [joi supports setting the default value](https://github.com/hapijs/joi/blob/v10.2.2/API.md#anydefaultvalue-description) as well, maybe the coalescing could be removed here and do it in validate.js